packages/devtools/pack-extension.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+const path = require('path');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const zip = require('bestzip');
+
+const DIST_PATH = path.resolve(__dirname, '../../dist/packages/devtools');
+
+if (!fs.existsSync(DIST_PATH)) {
+  console.error(`"${DIST_PATH}" does not exist, please run "build" command first`);
+  process.exit(1);
+}
+
+// Syncs the version from "package.json" with "manifest.json"
+const actualVersion = require('./package.json').version;
+
+const manifestPath = path.resolve(DIST_PATH, 'manifest.json');
+const manifestJSON = require(manifestPath);
```

Comment:
instead of doing it here, it's also possible to use beachball post-publish hooks to bump the manifest so that it's always in sync with the package.json after release

Reply:
When I discuss with @layershifter, beachball hook was our initial thoughts. But we went for syncing json in pack-extension.js. From my understanding we did it this way because it's more straightforward and easy to test.